docs/step-by-step/kubeapps-on-tkg/step-3.md

```diff
@@ -1,138 +1,115 @@
-# Step 3 - Add Application Repositories
+## Step 3: Add Application Repositories to Kubeapps
 
```

Comment:
I feel like there is a step missing here. Shouldn't we show the user how to log in using the identity management?

Reply:
Mmm, that's true. At least some screenshots with the info. I have a couple if you want them

Reply:
Maybe you could add that briefly with the screenshots and I can check it?